ytfzf

```diff
@@ -151,6 +153,7 @@ basic_helpinfo () {
     printf "     -l, --loop           <search-query>    Loop: prompt selector again after video ends\n";
     printf "     -s, --search-again   <search-query>    After the video ends make another search \n";
     printf "     -L, --link-only      <search-query>    Prints the selected URL only, helpful for scripting\n";
+    printf "     -N, --notification   <0,1>  			Send notification when play video\n";
```

Comment:
It doesn't need a 0 or a 1 right?
We could just leave that part empty

Reply:
if you are making it in such a way that it takes it arguemnt, then use --notification=
and also for parse opt you would need to send the argument as `"$2"`, refer the other flags that are already been done.

Reply:
fixed on newer commit